packages/core/src/api.ts

```diff
@@ -486,7 +486,7 @@ export class API {
      * Revoke a [[Session]], effectively logging out any client authenticated with it
      */
     @Handler(String, undefined)
-    revokeSession(_id: SessionID): PromiseWithProgress<void> {
+    revokeSession(_params: { id: SessionID; revokedForFailedAttempts?: boolean }): PromiseWithProgress<void> {
```

Comment:
If you you want to use anything other than a primitive value as an argument, you'll have to define a parameter class for that and adjust the `@Handler` decorator accordingly. See https://github.com/padloc/padloc/blob/052477b6c908092742d602316073b5bb6f53cf5c/packages/core/src/api.ts#L449-L452 for an example.